REFERENCE.md

```diff
@@ -0,0 +1,363 @@
+# Learnosity Java-SDK: Reference guide
+
+## Usage
+
+### Init
+
+The Init class is used to create the necessary *security* and *request* details used to integrate with a Learnosity API. Most often this will be a JavaScript object.
+
+The Init constructor takes either 3 arguments:
+
+ * String  service type
+ * Object  security details (**no secret**)
+ * String  secret
```

Comment:
Codacy has a fix for the issue: [Expected: 0; Actual: 1](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
* String  secret
```